travis-ci/vmtest/run_selftests.sh

```diff
@@ -20,5 +31,13 @@ fi
 
 cd libbpf/selftests/bpf
 
-echo TEST_PROGS
-./test_progs ${BLACKLIST:+-b$BLACKLIST} ${WHITELIST:+-t$WHITELIST}
+set +e
+exitcode=0
+for test_func in test_progs test_maps test_verifier; do
+	${test_func}; c=$?
+	if [[ $c -ne 0 ]]; then
+		exitcode=$c
+	fi
+done
```

Comment:
there is `set -e` on line 3, not sure why you need this instead of just:

```
test_progs
test_maps
test_verifier
```

Reply:
This is done to run all of `test_progs`, `test_maps`, `test_verifier` even if any of them fails. That's why `set +e` is set.

Reply:
Ah, I see. I think it's going to be painful in practice, though, because if test_progs fails, but subsequent test_maps and test_verifiers passes, it will be a pain to find the failure. Let's keep -e and run them one by one until first failure?

Reply:
ok, will change in a follow-up.